api/src/modules/import-data/import-data.controller.ts

```diff
@@ -35,10 +32,8 @@ export class ImportDataController {
   async importSourcingRecords(
     @UploadedFile() xlsxFile: Express.Multer.File,
   ): Promise<any> {
-    try {
-      return await this.importDataService.importSourcingRecords(xlsxFile.path);
-    } catch (err) {
-      throw new BadRequestException(err);
-    }
+    // TODO: Add userId once auth is in place
+    const userId: string = '2a833cc7-5a6f-492d-9a60-0d6d056923ea';
```

Comment:
This could be a good usage for a tiny type, but messes with some DTOs, entity itself and potentially a middleware we will have after guarding some controller/endpoints

Let's discuss further